storio-sqlite/src/test/java/com/pushtorefresh/storio/sqlite/operation/put/PutObjectsStub.java

```diff
@@ -106,18 +130,37 @@ static PutObjectsStub newPutStubForMultipleObjectsWithTypeMappingWithoutTransact
     }
 
     void verifyBehaviorForMultipleObjects(@NonNull PutResults<TestItem> putResults) {
+        // should be called once because of Performance!
+        verify(storIOSQLite).internal();
```

Comment:
I've used `verifyNoMoreInteractions()` so, it's okay to don't have `verify(..., never())` in this case.